packages/tests/esm/src/entities/codegen/AuthorCodegen.ts

```diff
@@ -37,8 +37,8 @@ import {
   type ValueFilter,
   type ValueGraphQLFilter,
 } from "joist-orm";
-import type { Context } from "../../context.ts";
-import { Author, authorMeta, Book, type BookId, type Entity, EntityManager, newAuthor } from "../entities.ts";
+import type { Context } from "../../context.js";
+import { Author, authorMeta, Book, type BookId, type Entity, EntityManager, newAuthor } from "../entities.js";
```

Comment:
The logic to allow importing .ts extensions is a nice feature. However, I wouldn't describe it as the "happy path" for ESM. The happy path for TS + ESM is to import using .js extensions, since it is compatible with both TypeScript and in the transpiled code.